article.html

```diff
@@ -0,0 +1,683 @@
+                                            </div>
+                                            <div class="container responsivegrid copy socio aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <div class="text quote-content">
+                                                    <div id="text-49e22e7256" class="cmp-text">
+                                                        <p>Copy</p>
+                                                    </div>
+                                                </div>
+                                                <div class="image logo-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_1715762034/container_1796181454/tabs/item_1669125273755/image_copy.coreimg{.width}.png/1669126312045/quote-img.png" data-asset="/content/dam/LeaderImpact/site-./assets/Vector_2.png" data-title="lady smiling" id="image-6bcf13ac84" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                                                      <img src="./assets/Vector_2.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="lady smiling">
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="container responsivegrid article-bottom aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div id="article-bottom-container" class="cmp-container">
+                            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                                <div class="container responsivegrid img-text-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div class="image img-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_1715762034/container_1796181454/tabs/item_1669125273755/image_copy.coreimg{.width}.png/1669126312045/quote-img.png" data-asset="/content/dam/LeaderImpact/site-./assets/Rectangle_726.png" data-title="lady smiling" id="image-6bcf13ac84" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                                          <img src="./assets/Rectangle_726.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="lady smiling">
```

Comment:
Images are supposed to appear full-width on mobile but right now they're inside the padding. Use negative margins and `calc()` to fix that.